stdlib/3/_ast.pyi

```diff
@@ -11,15 +11,30 @@ class AST:
     _fields: ClassVar[typing.Tuple[str, ...]]
     def __init__(self, *args: Any, **kwargs: Any) -> None: ...
     lineno: int
-    col_offset: int
+    col_offset: int  # TODO: Not all nodes have this
+    if sys.version_info >= (3, 8):
+        end_lineno: int
+        end_col_offset: int  # TODO: Not all nodes have this
+        type_comment: Optional[str]  # TODO: Only put this where it exists
 
 class mod(AST):
     ...
 
+if sys.version_info >= (3, 8):
+    class type_ignore(AST): ...
+
+    class TypeIgnore(type_ignore): ...
+
+    class FunctionType(mod):
+        argtypes = ...  # type: typing.List[expr]
+        returns = ...  # type: expr
+
 class Module(mod):
     body = ...  # type: typing.List[stmt]
     if sys.version_info >= (3, 7):
         docstring: Optional[str]
+    if sys.version_info >= (3, 8):
+        type_ignores: typing.List[TypeIgnore]
```

Comment:
`type_ignore` seems to be more precise.

Reply:
No, it always constructs a `TypeIgnore` instance. IIUC the lowercase classes (`mod`, `expr` etc.) exist to represent non-terminal symbols in the grammar, but the actual objects returned are always instances of a specific terminal symbol.

Reply:
More precise was the wrong choice of words. Looking at the other classes, fields nearly always seem to accept non-terminals, instead of concrete terminal symbols. Which makes sense, because there are multiple potential terminals, which is not the case here. I guess using `TypeIgnore` is more convenient to the use, even if it's a bit inconsistent.

Reply:
Also, we have `List`'s invariant behavior. Anyway, thanks for the merge!